package.json

```diff
@@ -70,7 +70,7 @@
     "eslint-plugin-prettier": "3.1.2",
     "eslint-plugin-react": "7.18.3",
     "gh-pages": "2.2.0",
-    "husky": "3.1.0",
+    "husky": "4.2.3",
```

Comment:
Can you keep bumping versions out of components work ? Easier to revert in case of 💥

Reply:
I think this PR is no longer in sync with `v3` as I synced that branch with `master` now.
We'll need to clean up the history 👍🏼